data_structures/dict_text_analysis_test.py

```diff
@@ -0,0 +1,25 @@
+# test cases dictionary example to count words in a file.
+
+
+import pytest
+from dict_text_analysis import text_analysis
+import os
+
+
+@pytest.mark.parametrize(
+    "file_name, result",
+    [
+        (
+            "/test1.txt",
+            {"#": 1, "test": 2, "ya": 1},
+        ),
+        (
+            "/test2.txt",
+            {"#": 1, "test": 2, "check": 1, "this": 1, "out": 1, "ya": 1},
+        ),
+    ],
+)
+def test_dict_text_analysis(file_name, result):
+    file_name = os.path.dirname(__file__) + file_name
```

Comment:
No... this is not OS agnostic. Might work on linux, but will fail on windows.

Look up `os.path.join`.

Reply:
os.path.join is use instead of concatenating two strings